src/NGAC/ngac_types/resource.py

```diff
@@ -0,0 +1,53 @@
+"""
+Resource abstraction
+
+A resource can be a file or something else
+"""
+
+
+def relative_import(package, module):
+    """
+    Solves pythons shitty relative import problem
+    """
+    import os
+
+    exec(
+        f"""from {"" if os.getcwd().endswith("ngac_types") else "ngac_types." if os.getcwd().endswith("NGAC") else "NGAC.ngac_types."}{package} import {module}"""
+    )
+    return locals()[module]
+
+
+NgacObject = relative_import("ngac_object", "NgacObject")
+OA = relative_import("ngac_attribute", "ObjectAttribute")
+
+from typing import List

```

Comment:
I agree that it's muuuch better to do it that way, however since python is python or since I don't know it well enough, the from ... import ... syntax breaks when running from different root folders.

So running the script as a submodule would change the paths to the files we want to import things from. If there is a better way to make it redundant to this problem I would very much prefer it. 

Original implementation is something like
```python
from ngac_object import NgacObject
from ngac_attribute import ObjectAttribute as OA
```
both of these files are in the same folder as the code file in question. But still it seems to break when doing something like
```python
# Not the path to resource is ./ngac_types/resource.py
from ngac_types.resource import Resource
```
This import would break the relative imports in resource.py

If there is some better way of doing this I'd gladly do that, I don't like using exec

Reply:
Okey, Then it makes sense to have the std lib import (or if its a pip lib) where it is as well. I usually get problems like this pretty often that I resolve with a "run-file" this way all the code is always executed from the same spot. But that not a optimal solution either. So I guess sticking with this is cleaner.